lib/fs_driver.py

```diff
@@ -54,7 +55,7 @@ def fs_read_cb(drv, fs_file, buf, btr, br):
     return lv.FS_RES.OK
 
 
-def fs_seek_cb(drv, fs_file, pos):
+def fs_seek_cb(drv, fs_file, pos, whence):
     try:
         # to =
         fs_file.cast()['file'].seek(pos, 0)
```

Comment:
`fs_seek_cb` recieves `whence` argument but ignores it.
Did you try to provide it to `seek`?
Related: https://github.com/lvgl/lvgl/issues/1763#issuecomment-851897110

Reply:
I had just added "whence" because I got an error if I didn't, and then forgot to have a look to find out what it is good for. I now added it to seek as you proposed (and that is where it should go). 
The other modifications have also been done. Sorry, I could have seen these **before** sending the PR.